aspnet/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/ViewModels/StormSessionViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace TestingControllersSample.ViewModels
+{
+    public class StormSessionViewModel
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public DateTime DateCreated { get; set; }
```

Comment:
Should genereally use DateTimeOffset, it's rare that DateTime is the correct type.